Revit_Core_Engine/Create/View/ViewPlan.cs

```diff
@@ -97,6 +97,77 @@ public static View ViewPlan(this Document document, Autodesk.Revit.DB.Level leve
         }
 
         /***************************************************/
+
+        [Description("Creates and returns a new Floor Plan view in the current Revit file.")]
+        [Input("document", "Revit current document to be processed.")]
+        [Input("level", "The level that the created Floor Plan refers to.")]
+        [Input("viewName", "Optional, name of the new view.")]
```

Comment:
```suggestion
        [Input("viewName", "(Optional) Name of the new view.")]
```

Reply:
View Name is not optional - I changed it